components/menu/index.css

```diff
@@ -291,17 +291,46 @@
 	}
 }
 
-.spectrum-Menu-itemIcon {
+.spectrum-Menu-itemIcon.spectrum-Icon {
 	fill: var(--highcontrast-menu-item-color-default, var(--mod-menu-item-label-icon-color-default, var(--spectrum-menu-item-label-icon-color-default)));
 	color: var(--highcontrast-menu-item-color-default, var(--mod-menu-item-label-icon-color-default, var(--spectrum-menu-item-label-icon-color-default)));
 }
 
-.spectrum-Menu-checkmark {
-	align-self: center;
```

Comment:
This is overwritten by later CSS in the file that sets it to `align-self: start;`